testdata-generator-common/src/main/java/io/openepcis/testdata/generator/format/RandomDateTimeGenerator.java

```diff
@@ -44,11 +44,7 @@ public OffsetDateTime nextDate() {
           .toOffsetDateTime();
     }
     try {
-      return Instant.ofEpochMilli(
-              randomData.nextLong(
-                  fromTime.toInstant().toEpochMilli(), toTime.toInstant().toEpochMilli()))
-          .atZone(ZoneId.systemDefault())
-          .toOffsetDateTime();
+      return Instant.ofEpochMilli(randomData.nextLong(fromTime.toInstant().toEpochMilli(), toTime.toInstant().toEpochMilli())).atZone(ZoneId.systemDefault()).toOffsetDateTime();
```

Comment:
I am not 100% sure if this how we would want it, maybe we should add support for custom timezone in input template in the future.

Reply:
https://github.com/openepcis/epcis-testdata-generator/issues/37